src/translations/en.json

```diff
@@ -0,0 +1,33 @@
+{
+  "footer": {
+    "discover_html": "Discover all about Paratii on %{homePageLink}",
+    "homePage": "paratii.video",
+    "betaTool_html": "v0.0.1: This is a beta tool! We &hearts; to get feedback on Telegram %{telegramBrazilLink} or %{telegramEnglishLink}",
```

Comment:
I have implemented a solution for #689. If we add an affix of `_html` to our keys, then the `<TranslatedText />` component will convert all passed in options, even if they contain JSX, to raw strings and inject them (unsafely) as html. 

We cannot use `_html` translations with anything that contains user input.